Classes/AttachPermissionsToGroups.php

```diff
@@ -84,7 +84,11 @@ private function expandGroupPermissionsWithPermissionSet(array $group, Permissio
                 if (MathUtility::canBeInterpretedAsInteger($siteOrPage)) {
                     $finalSitesAndPages[] = $siteOrPage;
                 } else {
-                    $finalSitesAndPages[] = GeneralUtility::makeInstance(SiteFinder::class)->getSiteByIdentifier($siteOrPage)->getRootPageId();
```

Comment:
at some point we can get rid of the makeInstances everywhere

Reply:
done